css/style.css

```diff
@@ -0,0 +1,73 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+/* body{
+    display: block;
+} */
+
+#navbar{
+    /* border: 2px solid red; */
+    position: fixed;
+    left: 0;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    min-width: 250px;
+    overflow: auto;
+    border-right: 2px solid black;
+    background-color: white;
+}
+
+header{
+    padding: 0.5em 0 0.5em 1em;
+}
+
+.nav-link{
+    text-decoration: none;
+    color: black;
+    text-align: left;
+    border-top: 2px solid black;
+    padding: 0.5em 0 0.5em 1em;
+    margin-top: 0.5em;
+}
+
+main{
+    margin-left: 260px;

```

Comment:
- [ ] Please try to be more professional in your comments and add solid reasons why there is the problem in the `margin` Kindly look at the point below, which will help you perform code reviews very well but please don't use my example.
   - Kindly fix your `margin` on left hand-side, as I can see your text is going outside the screen from the left hand-side which makes poor readeability and cause *Bad User Experience*.  

## Your comment

>  Please fix the margin error 😸